### PR TITLE
DOCSP-34486: v6.3 release

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-34486>
Staging:
- [Whats new](https://preview-mongodbrustagir.gatsbyjs.io/node/DOCSP-34486-node-v6.3/whats-new/#what-s-new-in-6.3)
- [compat](https://preview-mongodbrustagir.gatsbyjs.io/node/DOCSP-34486-node-v6.3/compatibility/)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/compatibility.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ Compatibility
    :values: reference
 
 .. meta:: 
-   :keywords: node.js
+   :keywords: node.js, driver v3.4, driver v4.3, driver v4.4, driver v4.5, driver v4.6, driver v4.9, driver v4.10, driver v4.11, driver v4.12, driver v4.13, driver v4.14, driver v4.15, driver v4.16, driver v5.8, driver v6.1, driver v6.2
```
Comment (rustagir) [inline source/compatibility.txt:12]: Note: these are the versions hidden after collapsing the server compat table

File: source/whats-new.txt [hunk 2]
```diff
@@ -53,6 +54,43 @@ Learn what's new in:
...
+     new MongoClient('<connection string>', { serverMonitoringMode: 'stream' });
+
+- Fixes a connection leak when the ``serverApi`` client option is set.
+  In previous versions, the driver sent a legacy command to the server
+  that returned an error, and the error-handling logic didn't correctly
+  remove the socket causing the connection leak.
+
+- Deprecates the ``contentType`` and ``aliases`` GridFS options. To
+  store this information, add corresponding fields to the ``metadata``
+  document.
```
Comment (mongoKart) [inline source/whats-new.txt:88]: S: clarity

```suggestion
- Deprecates the ``contentType`` and ``aliases`` GridFS options. To
  store the content type and aliases, add ``contentType`` and ``aliases``
  fields to the ``metadata`` document.
```

File: source/whats-new.txt [hunk 3]
```diff
@@ -53,6 +54,43 @@ Learn what's new in:
...
+  streaming mode.
+
+  You can set the ``serverMonitoringMode`` option in a
+  ``MongoClientOptions`` instance or as a connection string option. The
+  following example shows how to create a client with the option set to
+  ``stream``:
+  
+  .. code-block:: js
+     
+     new MongoClient('<connection string>', { serverMonitoringMode: 'stream' });
```
Comment (mongoKart) [inline source/whats-new.txt:79]: S: I would probably leave most or all of this info to be added in the docs ticket for this feature, but it doesn't hurt to leave it in.

File: source/whats-new.txt [hunk 4]
```diff
@@ -53,6 +54,43 @@ Learn what's new in:
...
 
+.. _version-6.3:
+
+What's New in 6.3
+-----------------
+
+The {+driver-short+} v6.3 release includes the following features:
+
+- Adds the ``serverMonitoringMode`` client option to control the
+  behavior of the monitoring connection between each node in a topology.
```
Comment (mongoKart) [inline source/whats-new.txt:65]: I: [between/among](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#:~:text=between-,When%20referring%20to,-three%20or%20more)

```suggestion
  behavior of the monitoring connection among the nodes [or "between each pair of nodes"; i'm not sure which is correct] in a topology.
```

File: source/whats-new.txt [hunk 5]
```diff
@@ -53,6 +54,43 @@ Learn what's new in:
...
+  This option takes a value of ``auto`` (default), ``poll``, or
+  ``stream``. When the setting is ``auto``, the environment the driver
+  runs in determines the monitoring mode. Function-as-a-service (FaaS)
```
Comment (mongoKart) [inline source/whats-new.txt:68]: S: I would push for passive voice here for clarity

```suggestion
  ``stream``. When the setting is ``auto``, the monitoring mode is
   determined by the environment where the driver is running. Function-as-a-service (FaaS)
```

File: source/whats-new.txt [hunk 6]
```diff
@@ -53,6 +54,43 @@ Learn what's new in:
...
+  environments prefer the polling mode and all others prefer the
+  streaming mode.
```
Comment (mongoKart) [inline source/whats-new.txt:70]: S: "prefer" doesn't seem quite right and is pretty [anthropomorphic](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#don-t-humanize-inanimate-objects)

```suggestion
  runs in determines the monitoring mode. The driver uses polling
  mode in function-as-a-service (FaaS)
  environments and streaming mode everywhere else.
```

**Review by mongoKart — APPROVED** [review]
some suggestions but lgtm otherwise

File: source/whats-new.txt (hunk 3 above)
Comment (rustagir) [reply source/whats-new.txt:79]: I can take this out of the whats new and add it to the connection options entry (in this PR as well) instead.

**Pull request merged**